@stellar/freighter-api/src/index.ts

```diff
@@ -5,3 +5,10 @@ import { getNetwork } from "./getNetwork";
 
 export { getPublicKey, signTransaction, isConnected, getNetwork };
 export default { getPublicKey, signTransaction, isConnected, getNetwork };
+
+window.freighterApi = {
+  getPublicKey,
+  signTransaction,
+  isConnected,
+  getNetwork,
+};
```

Comment:
this is the big change to the API. 

cdnjs will package this file and host on their server, so we will expose both the es modules for npm users as well as hanging the methods off of `window` for other apps